packages/models/src/users.ts

```diff
@@ -167,7 +167,7 @@ export class Users {
     return this.from(row);
   }
 
-  from(row: IUser.Row): IUser.Self {
+  async from(row: IUser.Row): Promise<IUser.Self> {
```

Comment:
We don't need this `async`. Please remove it and update all other places accordingly.